### PR TITLE
[11.0][MIG] mgmtsystem nonconformity

### PR DESCRIPTION
**Pull request by jalzaga**
Module mgmtsystem_nonconformity Migration to version 11.0 #200

### CODE REVIEW

**Comment by ivantodorovich** [comment]
pls remove old migration scripts

**Comment by ivantodorovich** [comment]
This PR only needs the removal of migration scripts and a rebase would be nice to make runbot and travis happy

**Comment by ivantodorovich** [comment]
@jalzaga can you rebase, to make travis happy?

File: mgmtsystem_nonconformity/README.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Enter comments into the input field below the "Plan Review" section, those comments are required to reach the next stage.
+
+To begin the work on the planned Actions change the stage of the NC to open by clicking on the "In Progress" button in the top right corner.
+
+When all actions of the plan are done, their effectiveness must be evaluated
+before closing the NC.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/128/10.0
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity/README.rst]: update to 11.0

File: mgmtsystem_nonconformity/README.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/Management-system/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+Management-system/issues/new?body=module:%20
+mgmtsystem_system%0Aversion:%20
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity/README.rst]: 11.0

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
update readme

**Comment by jalzaga** [comment]
Done @ivantodorovich

**Comment by ivantodorovich** [comment]
Please rebase, since this module depends on mgmtsystem_action.
That should make travis and runbot happy :)

**Comment by jalzaga** [comment]
I have already done the rebase @ivantodorovich . Is there anything else missing?

**Comment by jalzaga** [comment]
I have already done the rebase @ivantodorovich . Is there anything else missing?

**Comment by jalzaga** [comment]
I have already done the rebase @ivantodorovich . Is there anything else missing?

**Comment by max3903** [comment]
@jalzaga Travis and Runbot are red. You still need to fix those errors. Thanks!

File: mgmtsystem_nonconformity/tests/test_cause.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        """Do a regular assertRaises but perform rollback at the end
+        """
+        with self.assertRaises(*args, **kwargs) as ar:
+            yield ar
+        self.cr.rollback()
+
+    def test_create_cause(self):
+        with self.assertRaisesRollback(IntegrityError):
+            # Will generate an error in the logs but we handle it
+            self.env['mgmtsystem.nonconformity.cause'].create({})
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity/tests/test_cause.py]: This line

File: mgmtsystem_nonconformity/tests/test_origin.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+##############################################################################
+from odoo.tests import common
+from psycopg2 import IntegrityError
+
+
+class TestModelOrigin(common.TransactionCase):
+    def test_create_origin(self):
+
+        with self.assertRaises(IntegrityError):
+            self.env['mgmtsystem.nonconformity.origin'].create({})
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity/tests/test_origin.py]: and this line

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
I think those lines are the ones making travis and runbot red.
Those test cases make some database errors to appear in the logs. 
I would just remove them, what do you think @max3903 ?

**Comment by max3903** [comment]
@ivantodorovich :+1:

**Comment by ivantodorovich** [comment]
Thanks for that commit @jalzaga 

Travis is complaining because of the now unused IntegrityError. Please remove it from the imports aswell.
```
======== Testing test_flake8 ========
mgmtsystem_nonconformity/tests/test_cause.py:22:1: F401 'psycopg2.IntegrityError' imported but unused
mgmtsystem_nonconformity/tests/test_origin.py:21:1: F401 'psycopg2.IntegrityError' imported but unused
```

Runbot is still complaining, becase of this line:
```
2018-10-25 15:29:03,782 165 WARNING openerp_test odoo.models: ir.cron.create() includes unknown fields: function, model
```
It's a migration error in the `mgmtsystem_action` module. `ir.cron` wasn't migrated properly.
It is fixed on this commit: https://github.com/OCA/management-system/pull/236/commits/200f73f10f4fb2bed72161ce1569375a473be8cd (PR https://github.com/OCA/management-system/pull/236)

Since it's unrelated to this PR, I would just ignore it. But, please, do remove the imported IntegrityError

**Comment by jalzaga** [comment]
Done @ivantodorovich

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Thanks!
LGTM 👍 

If you can review https://github.com/OCA/management-system/pull/236, among other things, it fixes the `ir.cron` in `mgmtsystem_action` that's causing runbot to complain.

**Review by max3903 — APPROVED** [review]

**Pull request merged**